src/infuse.js

```diff
@@ -195,7 +195,11 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
                     }
                 }
             }
-            return undefined;
+            if (this.throwOnMissing) {
```

Comment:
Why using the option ```throwOnMissing``` here? ```getMapping``` is not used internally, it should never throw an error.